[2023-08-01 09:52:53,941] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.weather_data_to_BQ scheduled__2023-07-31T09:52:26.300414+00:00 [queued]>
[2023-08-01 09:52:53,967] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.weather_data_to_BQ scheduled__2023-07-31T09:52:26.300414+00:00 [queued]>
[2023-08-01 09:52:53,967] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 09:52:53,967] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-01 09:52:53,968] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 09:52:54,058] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): weather_data_to_BQ> on 2023-07-31 09:52:26.300414+00:00
[2023-08-01 09:52:54,065] {standard_task_runner.py:52} INFO - Started process 153 to run task
[2023-08-01 09:52:54,069] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_BigQuery', 'weather_data_to_BQ', 'scheduled__2023-07-31T09:52:26.300414+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpln5e8o8g', '--error-file', '/tmp/tmpw_wxtjaq']
[2023-08-01 09:52:54,070] {standard_task_runner.py:80} INFO - Job 145: Subtask weather_data_to_BQ
[2023-08-01 09:52:54,180] {task_command.py:370} INFO - Running <TaskInstance: s3_to_BigQuery.weather_data_to_BQ scheduled__2023-07-31T09:52:26.300414+00:00 [running]> on host 3548e63a091e
[2023-08-01 09:52:54,413] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_BigQuery
AIRFLOW_CTX_TASK_ID=weather_data_to_BQ
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T09:52:26.300414+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T09:52:26.300414+00:00
[2023-08-01 09:52:56,068] {logging_mixin.py:115} INFO - Error: local variable 'data' referenced before assignment
[2023-08-01 09:52:56,072] {python.py:173} INFO - Done. Returned value was: None
[2023-08-01 09:52:56,203] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=s3_to_BigQuery, task_id=weather_data_to_BQ, execution_date=20230731T095226, start_date=20230801T095253, end_date=20230801T095256
[2023-08-01 09:52:56,296] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-01 09:52:56,424] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
